src/modules/clipboard/clipboardtempmode.cpp

```
@@ -0,0 +1,128 @@
+    auto *inputContext = keyEvent.inputContext();
+    if (keyEvent.isRelease()) {
+        return false;
+    }
+
+    auto candidateList = inputContext->inputPanel().candidateList();
+    if (candidateList) {
+        int idx = keyEvent.key().digitSelection();
+        if (idx >= 0) {
+            if (idx < candidateList->size()) {
+                candidateList->candidate(idx).select(inputContext);
+            }
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If FCITX_CONFIGURATION is a macro then please configure it.

(unknownMacro)